packages/core-commands/src/index.js

```diff
@@ -1 +1,2 @@
+export { unlock } from './lock-unlock';
```

Comment:
Unfortunately this is not something we should export but it makes "private APIs" public for any third-party plugin and this shouldn't be the case. 

Maybe we should just export the CommandWrapper component like we used to do? 🤔

Reply:
I exported a commands-menu-wrapper component from the core-commands packages and used a small inline script to register it on the page in [d381a0d](https://github.com/WordPress/gutenberg/pull/54515/commits/d381a0d2549e409315172f4f205caad54dc1210d). I honestly would have preferred to this wrapper in the commands package instead but the circular dependency doesn't make this simple.